playbooks/ansible.cfg

```diff
@@ -10,4 +10,6 @@ roles_path=../../ansible-roles/roles:../../ansible-private/roles:../../ansible-r
 ansible_managed=This file is created and updated by ansible, edit at your peril
 
 [ssh_connection]
-ssh_args=-o ControlMaster=auto -o ControlPersist=60s -o ControlPath="~/.ansible/tmp/ansible-ssh-%h-%p-%r" -o ServerAliveInterval=30
+pipelining = True
```

Comment:
:raised_hands: 

As a side note, I usually have this variable on my `.bashrc`:
```
$ env | grep -i pipe
ANSIBLE_SSH_PIPELINING=1
```

Reply:
Yeah, I did the same, but it might as well just go in the ansible config so everyone gets the improvement whether they know about it or not.